translations/messages.fr.xlf

```diff
@@ -0,0 +1,3065 @@
+        To do this, you have to click on the crossed circle that appears at the center of the board when it is possible.&lt;br/&gt;&lt;br/&gt;
+        Do it.</source>
+        <target>Chacun à son tour, les joueurs posent une pièce sur le plateau, et effectuent éventuellement une rotation d&apos;un quadrant.
+        Tant qu&apos;il existe des quadrants neutres, c&apos;est à dire des quadrants qui ne changeraient pas après avoir été tournés, l&apos;option de ne pas effectueur de rotation est acceptée.
+        Pour ce faire il faut cliquer sur le rond barré qui apparaît au centre du plateau quand c&apos;est possible.&lt;br/&gt;&lt;br/&gt;
+        Faites-le.</target>
+      </trans-unit>
+      <trans-unit id="1640662905904405955" datatype="html">
+        <source>You have made a move with a rotation. This tutorial step is about moves without rotations!</source>
+        <target>Vous avez effectué un mouvement avec rotation, cette étape du didacticiel concerne les tours sans rotations !</target>
+      </trans-unit>
+      <trans-unit id="8330321104835134748" datatype="html">
+        <source>Move with rotation</source>
+        <target>Mouvement avec rotation</target>
+      </trans-unit>
+      <trans-unit id="1712930066423533652" datatype="html">
+        <source>After putting a piece, arrows will appear on non-neutral blocks.&lt;br/&gt;&lt;br/&gt;
+        Click on one of them and see the rotation!</source>
+        <target>Après avoir déposé sa pièce, des flèches apparaîtront sur les quadrants non neutres.&lt;br/&gt;&lt;br/&gt;
+        Cliquez sur l&apos;une d&apos;entre elles et voyez la rotation !</target>
+      </trans-unit>
+      <trans-unit id="5427363142376983767" datatype="html">
+        <source>You made a move without rotation, try again!</source>
+        <target>Vous avez effectué un mouvement sans rotation, recommencez !</target>
+      </trans-unit>
+      <trans-unit id="4197928800633217955" datatype="html">
+        <source>Congratulations! Note that if all blocks are neutral after you have put your piece, there will be no rotation.</source>
+        <target>Bravo ! Note : si tout les quadrants sont neutres après que vous ayez déposé votre pièce, il n&apos;y aura pas de rotation !</target>
+      </trans-unit>
+      <trans-unit id="682762602217958961" datatype="html">
+        <source>You must move your pieces upward.</source>
+        <target>Vous devez déplacer vos pièces vers le haut.</target>
+      </trans-unit>
+      <trans-unit id="1573455518890855659" datatype="html">
+        <source>You must start your move with one of your pieces.</source>
+        <target>Votre mouvement devrait commencer avec une de vos pièces !</target>
+      </trans-unit>
+      <trans-unit id="155508093542570099" datatype="html">
+        <source>Your move must have supporting pieces.</source>
+        <target>Votre déplacement doit avoir des pièces qui le supportent.</target>
+      </trans-unit>
```

Comment:
Connaissant les gens, "Votre pièce doit atterrir sur le plateau ou sur 4 autres pièces" est bien moins ambigu, tu penses pas ?

Reply:
oublié celui-ci, j'le laisse pour la passe suivante ! Mais sinon je suis d'accord.
Et "Your move must land on the board or on 4 other pieces"

Reply:
(fait localement)